samples/snippets/fsharp/azure/blob-storage.fsx

```diff
@@ -29,88 +25,67 @@ let storageAccount = CloudStorageAccount.Parse(storageConnString)
 let localFile = __SOURCE_DIRECTORY__ + "/myfile.txt"
 File.WriteAllText(localFile, "some data")
 
-//
-// Create the blob service client.
-//
-
-let blobClient = storageAccount.CreateCloudBlobClient()
-
 //
 // Create a container.
 //
 
- // Retrieve a reference to a container.
-let container = blobClient.GetContainerReference("mydata")
+ // Create a blob container client.
+let container = BlobContainerClient(storageConnString, "myContainer");
 
 // Create the container if it doesn't already exist.
 container.CreateIfNotExists()
 
-let permissions = BlobContainerPermissions(PublicAccess=BlobContainerPublicAccessType.Blob)
-container.SetPermissions(permissions)
+let permissions = PublicAccessType.Blob
+container.SetAccessPolicy(permissions)
 
 //
 // Upload a blob into a container.
 //
 
 // Retrieve reference to a blob named "myblob.txt".
-let blockBlob = container.GetBlockBlobReference("myblob.txt")
+let blockBlob = container.GetBlockBlobClient("myblob.txt")
```

Comment:
We should prefer `let blob = container.GetBlobClient("myblob.txt")`.  That will still allow you to upload a block blob, but you don't get bothered by any of the block list methods, etc.